torch/_dynamo/symbolic_convert.py

```diff
@@ -706,6 +706,9 @@ def LOAD_GLOBAL(self, inst):
                 assert name in self.f_builtins
                 self.exec_recorder.builtins[name] = self.f_builtins[name]
 
+        if inst.argval == "AssertionError":
```

Comment:
Is the reasoning that even after converting assert into torch._assert, we still see another assertion that hasn't been addressed?

Reply:
yes, because ```_detect_and_normalize_assert_statement``` can only detect/apply to the pattern we supported. For what we don't support(e.g, non-string message), it goes with regular path.

Reply:
I see makes sense.